.github/workflows/hive-tests.yml

```diff
@@ -10,13 +10,12 @@ on:
         description: Test suite
         type: choice
         options:
-          - ''
+          - 'all'
           - devp2p
           - ethereum/consensus
           - ethereum/eest
           - ethereum/engine
           - ethereum/graphql
-          - ethereum/rpc
```

Comment:
If they are the same, maybe removing `rpc-compat` instead, as simple `rpc` reads better? Or is there a reason?